src/main.cc

```diff
@@ -157,6 +157,10 @@ void register_env_vars()
             "window_height", false,
             [](StringView name, const Context& context) -> String
             { return to_string(context.window().dimensions().line); }
+        }, {
+            "server_pid", false,
+            [](StringView name, const Context& context) -> String
+            { return to_string(Server::instance().pid()); }
```

Comment:
It would issue a syscall everytime the variable is expanded, since it's not going to change I think storing it is cleaner.